### PR TITLE
Allow fromBeginning config in kafka trigger node

### PR DESCRIPTION
**Pull request by p1anv**

### CODE REVIEW

File: packages/nodes-base/nodes/Kafka/KafkaTrigger.node.ts [hunk 1]
```diff
@@ -141,7 +148,7 @@ export class KafkaTrigger implements INodeType {
 
 		await consumer.connect();
 
-		await consumer.subscribe({ topic, fromBeginning: true });
+		await consumer.subscribe({ topic, fromBeginning: (options.fromBeginning)? true : false });
```
Comment (MedAliMarz) [inline packages/nodes-base/nodes/Kafka/KafkaTrigger.node.ts:151]: Please make sure, that the `options` variable is assigned before using one of its fields.

**Review by MedAliMarz — COMMENTED** [review]
Hey @planvin , Thank you for your contribution, just a minor changes needed to make the `fromBeginning` option enabled. Also, it would be cool if you can provide a workflow that uses such functionality.

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with n8n@0.130.0

**Pull request merged**